A quick, Cygwin-only hack to build the one plugin. A complete patch would 
require an AM_CONDITIONAL(PLATFORM_WIN32) plus installing the implib
and adding it to xfmedia-plugin.pc.

Unfortunately, infopipe causes the program to crash, so we don't ship with this.

--- origsrc/xfmedia-0.9.2/Makefile.am	2006-11-27 14:14:56.000000000 -0600
+++ src/xfmedia-0.9.2/Makefile.am	2009-07-15 11:41:23.986398900 -0500
@@ -3,9 +3,9 @@
 SUBDIRS = \
 	pixmaps \
 	xfmedia \
+	src \
 	plugins \
 	po \
-	src \
 	doc
 
 appsdir = $(datadir)/applications
--- origsrc/xfmedia-0.9.2/plugins/infopipe/Makefile.am	2006-11-27 01:21:21.000000000 -0600
+++ src/xfmedia-0.9.2/plugins/infopipe/Makefile.am	2009-07-15 11:22:00.828575200 -0500
@@ -13,6 +13,8 @@ xfmedia_infopipe_la_CFLAGS = \
 	@GTK_CFLAGS@
 
 xfmedia_infopipe_la_LDFLAGS = \
+	-Wl,$(top_builddir)/src/libxfmedia.a \
+	-no-undefined \
 	-module \
 	-export-dynamic \
 	-avoid-version
--- origsrc/xfmedia-0.9.2/src/Makefile.am	2006-11-27 08:44:03.000000000 -0600
+++ src/xfmedia-0.9.2/src/Makefile.am	2009-07-15 11:21:23.561755800 -0500
@@ -64,6 +64,7 @@ xfmedia_CFLAGS = \
 
 xfmedia_LDFLAGS = \
 	-export-dynamic \
+	-Wl,--export-all-symbols,--out-implib,libxfmedia.a
 	@LIBX11_LDFLAGS@
 
 xfmedia_LDADD = \
